lib/sparql.jison

```diff
@@ -266,6 +266,25 @@
     }
     return merged;
   }
+
+  function getBindedVarsFromGroupGraphPattern(pattern) {
+    if (pattern.triples) {
+      const bindedVars = [];
+      for (let triple of pattern.triples) {
+        if (triple.subject.id.startsWith('?')) bindedVars.push(triple.subject.id);
+        if (triple.predicate.id.startsWith('?')) bindedVars.push(triple.predicate.id);
+        if (triple.object.id.startsWith('?')) bindedVars.push(triple.object.id);
```

Comment:
We probably want an `isVariable` function, and that one should likely be `term.termType === 'Variable'` (`id` is an implementation-specific feature, another term library might be injected)